p2p/proto/consensus.proto

```diff
@@ -1,42 +1,121 @@
 syntax = "proto3";
 
-import "p2p/proto/common.proto";
-import "p2p/proto/transaction.proto";
-import "p2p/proto/state.proto";
+import "common.proto";
 import "google/protobuf/timestamp.proto";
 
+package consensus;
+
 
 // WIP - will change
 
-message Proposal {
-    uint64                    block_number      = 2;
-    uint32                    round             = 3;
-    uint32                    pol               = 4;  // proof of lock
-    Hash                      block_header_hash = 5;
-    google.protobuf.Timestamp timestamp         = 6;
-    ConsensusSignature        signature         = 7;
+message Transaction {
+    bytes elements = 1;
+}
+
+message TransactionBatch {
```

Comment:
Oh as a protection against a malicious validator "replaying" the proposal but lying about the content. Interesting.

Reply:
On second thought I am not certain we need this. If a malicious validator is meant to relay a proposal we are worried about:
1. Sending invalid content - the signature on `ProposalContent` protects against this.
2. Censoring content - the signature on `ProposalContent` does not protect against this.

The `block_signature` in the `ProposalFin` message should protect us against both of these scenarios. In which case the signature on `ProposalContent` is unneeded for the proposal as a whole. 

Maybe it is useful since a malicious validator cannot waste our resources with invalid `ProposalContent` (DoS attack), but they could still do other things to achieve a similar end (many proposals in parallel). I'm leaning back towards removing this and just keeping the signature on the proposal as a whole.